.github/workflows/release.yml

```diff
@@ -53,3 +53,20 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+      - name: Publish Solidity Library
+        if: steps.changesets.outputs.published == 'true'
+        uses: actions/github-script@v6
+        env:
+          PUBLISHED_PACKAGES: ${{ steps.changesets.outputs.publishedPackages }}
+        with:
+          script: |
+            const publishedPackages = JSON.parse(process.env.PUBLISHED_PACKAGES);
+            const contractsPackage = publishedPackages.find((pkg) => pkg.name === '@sphinx-labs/contracts');
+            if (contractsPackage) {
+              github.rest.git.createRef({
+                owner: context.repo.owner,
+                repo: context.repo.repo,
+                ref: `refs/heads/v${contractsPackage.version}`,
+                sha: context.sha
+              });
+            }
```

Comment:
I tested this by creating a temporary test workflow (which I've since removed from this branch). You can see the successful run [here](https://github.com/sphinx-labs/sphinx/actions/runs/7659291853).